lib/cli/scripts/init-aae-env.ts

```diff
@@ -568,6 +568,7 @@ async function deployWithPayload(currentAbsentApp: any, projectRelease: any, env
         security: currentAbsentApp.security,
         infrastructure: currentAbsentApp.infrastructure,
         variables: currentAbsentApp.variables,
+        enableLocalDevelopment: currentAbsentApp.enableLocalDevelopment,
```

Comment:
I don't know if I have to specify a default value to handle when enableLocalDevelopment field doesn't exist in lib/cli/scripts/resources.ts (please see https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/cli/scripts/resources.ts#L207)